app/views/provider/admin/messages/trash/index.html.slim

```diff
@@ -8,20 +8,24 @@
   // the whole explanation: https://github.com/3scale/system/issues/6275
   //.operations
   //  = fancy_button_to 'Empty the trash', empty_provider_admin_messages_trash_index_path, method: :delete, class: 'empty_trash'
-  table#messages.data
+  table class="data pf-c-table pf-m-grid-lg" role="grid" id="messages"
```

Comment:
What do you think about adding an `aria-label` here as in `inbox` and `outbox`?

```suggestion
  table class="data pf-c-table pf-m-grid-lg" role="grid" aria-label="Messages trash" id="messages"
```

Reply:
I didn't like the label much.. so I skipped it. I am not sure "Messages trash" is meaningful enough.